packages/react-composites/tests/browser/call/app/BaseApp.tsx

```diff
@@ -25,6 +25,8 @@ initializeIconsForUITests();
 export function BaseApp(props: { queryArgs: QueryArgs; callAdapter?: CallAdapter }): JSX.Element {
   const { queryArgs, callAdapter } = props;
 
+  console.log(`Loaded test app with args ${JSON.stringify(queryArgs)}`);
```

Comment:
Minor: Is this for debugging purposes? Is it ok to leave it in?

Reply:
Intended to stay. This just makes the test apps' console a bit more useful.